packages/interpreter/src/components/scope.ts

```diff
@@ -35,7 +35,7 @@ export default class Scope {
   isContinueStatement() {
     return this._isContinueStatement;
   }
-
+  
```

Comment:
Please remove this change. Although it just an empty line change. We shouldn't see this commit on Scope file for clear visibility.

Reply:
Makes sense. Done.